server/lib/models/channel.go

```diff
@@ -53,8 +78,30 @@ func ListChannel(dbif Dbif, req *proto.ChannelListRequest) ([]*proto.Model_Chann
 	} else {
 		limit = CHANNEL_FETCH_LIMIT
 	}
-	//TODO: respect category parameter of req.
-	chs, err := dbif.Select(Channel{}, dbm.Stmt("select * from %s.channels order by id desc limit $1"), limit)
+	tmp := make([]string, 0)
+	cond := ""
+	order_by := "id desc"
```

Comment:
__[gometalinter]__
* [golint] don't use underscores in Go names; var order_by should be orderBy